drivers/CoreLED/include/CoreLED.h

```diff
@@ -86,8 +109,10 @@ class CoreLED : public interface::LED
 	bool _is_color_shown = false;
 
 	std::array<RGB, NumberOfLeds> _colors;
+	std::array<uint8_t, NumberOfLeds> _brightness;
 
-	static constexpr uint8_t brightness = 0xFF;
+	static constexpr auto default_brightness = 0xF0;
+	static constexpr auto minimum_brightness = 0xE0;
```

Comment:
puisque c'est des constantes, il faut utiliser la règle de naming https://github.com/leka/styleguides/blob/master/cppguide.md#constant-names

```suggestion
	static constexpr auto kBrightnessDefault = 0xF0;
	static constexpr auto kBrightnessMinimum = 0xE0;
```